packages/create-cloudflare/src/helpers/poll.ts

```diff
@@ -22,7 +22,7 @@ export const poll = async (url: string): Promise<boolean> => {
 	const domain = new URL(url).host;
 	const s = spinner();
 
-	s.start("Waiting for DNS to propagate");
+	s.start("Waiting for DNS to propagate. This might take up to two minutes.");
```

Comment:
this will be mostly true for Pages but I'm not sure if it's worth calling that out. IMHO it'll just make the copy more noisy